tools/validate_configs/test_configs/hpc-cluster-simple-nfs-sql.yaml

```diff
@@ -42,32 +42,36 @@ deployment_groups:
       sql_instance_name: slurm-sql8
       tier: "db-f1-micro"
 
+  - id: compute-nodeset
+    source: ./community/modules/compute/schedmd-slurm-gcp-v6-nodeset
```

Comment:
nit: Remove the `./` from each source line. See L29 for example.

Reply:
Removed.